subiquity/server/controllers/drivers.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2018 Canonical, Ltd.
```

Comment:
minor: year to update :) 

But since it's 2022 now ; I'm not sure it's relevant.